common/action-manager.js

```diff
@@ -66,14 +66,17 @@ am.ACTIONS_WITH_SITES = new Set([
   "HIGHLIGHTS_LINKS_RESPONSE"
 ].map(type => am.type(type)));
 
-function Notify(type, data) {
+function Notify(type, data, meta) {
```

Comment:
nit: could you add some comments for the arguments here? Making it easier to read?